components/Nav/Nav.tsx

```diff
@@ -67,7 +67,17 @@ export const Nav = (): JSX.Element => {
               <DrawerHeader>
                 <DrawerTitle data-testid="title">Gridiron Survivor</DrawerTitle>
               </DrawerHeader>
-              <ul className="m-0 flex flex-col gap-4 p-0">
+              <ul className="m-0 flex flex-col gap-1 p-0">
+                <li>
+                  <a
+                    href="/account/settings"
+                    className="p-0 text-base font-normal text-muted-foreground underline underline-offset-4 hover:text-primary/90"
+                    onClick={() => setOpen(false)}
+                    data-testid="settings-link"
+                  >
+                    Settings
+                  </a>
+                </li>
```

Comment:
Q: I'd love to know your reasoning behind why it is an `<a>` tag and not a `<Link>` tag?

Reply:
@vmaineng good catch! it should have been a Link using the `next/link` package.  Rule of thumb is:

- use Next Link for internal routes
- use a tag for external links

✅ updated to use Next Link.